store.go

```diff
@@ -3308,6 +3308,9 @@ const defaultConfigFile = "/etc/containers/storage.conf"
 // DefaultConfigFile returns the path to the storage config file used
 func DefaultConfigFile(rootless bool) (string, error) {
 	if rootless {
+		if configHome := os.Getenv("XDG_CONFIG_HOME"); configHome != "" {
```

Comment:
Should this be documented in the containers.storage.conf man page?

Reply:
LGTM
I'd be OK with a man page addition, but I am OK without if it causes angst in others.

Reply:
yes, I've pushed a new version with a change for the man page